src/view/Showdialog.tsx

```diff
@@ -0,0 +1,70 @@
+import React, {useState} from "react";
+import {DialogData} from "../utils/typer";
+import Lenke from "nav-frontend-lenker";
+import {Undertittel} from "nav-frontend-typografi";
+import {HoyreChevron} from "nav-frontend-chevron";
+import {Checkbox, Textarea} from "nav-frontend-skjema";
+import {Hovedknapp} from "nav-frontend-knapper";
+import {HenvendelseList} from "./HenvendelseList";
+
+interface Props {
+    dialog: DialogData;
+}
+
+export function Showdialog(props: Props) {
+    const [value, setValue] =  useState('Skriv her');
```

Comment:
Dette vil medføre at tekstfeltet er preutfylt med innhold, noe som ikke er spesielt bra for brukeropplevelsen. 
Sannsynligvis kan dere oppnå ønsket effekt med å sette denne til en tom string, og legge til `placeholder`-attributtet på tekstfeltet.